### PR TITLE
Revamp

### PR DESCRIPTION
**Pull request by il8677**

### CODE REVIEW

File: game/main.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+import random
+
+
+class Biome:
+	name = "Void"
+	icon = "X"
+
+
+class Field(Biome):
```
Comment (il8677) [inline game/main.py]: Why does this exist? No additional functionality is added, no reason for polymorphism

File: game/main.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			for location in surroundings:
+				if self.map[location[1]][location[0]].name is not "Void":
+					print(f"Replacing {seed[1]}, {seed[2]} with {seed[0].name}")
+					self.map[location[1]][location[0]] = seed[0]()
+					self.seeds.append((seed[0], location[0], location[1]))
+
+
+
+	def generateTerrain(self):
+		self.map = [[Biome() for j in range(self.width)] for i in range(self.height)]
```
Comment (il8677) [inline game/main.py]: Nice lambada functions

File: game/main.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+				(seed[2], (seed[1] + 1) % self.height),  # below
+				((seed[2] - 1) % self.width, (seed[1]) % self.height),  # left
+				((seed[2] + 1) % self.width, (seed[1]) % self.height)  # right
+			]
+
+			for location in surroundings:
+				if self.map[location[1]][location[0]].name is not "Void":
+					print(f"Replacing {seed[1]}, {seed[2]} with {seed[0].name}")
+					self.map[location[1]][location[0]] = seed[0]()
+					self.seeds.append((seed[0], location[0], location[1]))
```
Comment (il8677) [inline game/main.py]: NEVER modify subject  of for loop inside for loop...

File: game/main.py (hunk 1 above)
Comment (testertester-123) [reply game/main.py]: Additional functionality is to be added later. What's more, we need different biomes if we want to generate different biomes...

File: game/main.py (hunk 2 above)
Comment (DyingEcho) [reply game/main.py]: Is this sarcasm?

**Pull request closed without merging**